src/test/safe.dom.settings.owners.test.js

```diff
@@ -10,7 +10,7 @@ import {
   checkRegisteredTxRemoveOwner,
   checkRegisteredTxReplaceOwner,
 } from './utils/transactions'
-import { SETTINGS_TAB_BTN_TEST_ID } from '~/routes/safe/components/Layout'
+import { SETTINGS_TAB_BTN_TEST_ID } from '~/routes/safe/components/Layout/Layout'
```

Comment:
`Layout/Layout` hurts me, can we do something about it?

Reply:
fixed